force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +430,12 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        if(person1.age == person2.age){
+            return person2;
+        }
+        else if(person1.age > person2.age){
+            return person1;
+        }
+        return person2; // Replace null with the variable you used to store the result
     }
```

Comment:
Can you simplify this to one If condition and then a return? If the values are tied, you can return them either.